addons/website/static/tests/tours/automatic_editor.js

```diff
@@ -0,0 +1,90 @@
+
+odoo.define('test_website_modules.tour.automatic_editor', function (require) {
+'use strict';
+
+const tour = require('web_tour.tour');
+
+tour.register('automatic_editor_on_new_website', {
+    test: true,
+    url: '/',
+},
+[
+    {
+        content: "Select the language dropdown",
+        trigger: '.js_language_selector .dropdown-toggle'
+    },
+    {
+        content: "click on Add a language",
+        trigger: 'a.o_add_language',
+    },
+    {
+        content: "Select dropdown",
+        trigger: 'select[name=lang]',
+        run: function () {
+            $('select[name="lang"]').val('"pa_GB"').change();
+        }
+    },
+    {
+        content: "load parseltongue",
+        extra_trigger: '.modal select[name="lang"]:propValueContains(pa_GB)',
+        trigger: '.modal-footer button:first',
+    },
+    {
+        content: "Select the language dropdown",
+        trigger: '.js_language_selector .dropdown-toggle',
+    },
+    {
+        content: "Select parseltongue",
+        trigger: 'a.js_change_lang[data-url_code=pa_GB]',
+    },
+    {
+        content: "Check that we're on parseltongue",
+        trigger: 'html[lang=pa-GB]',
+        run: function () {}, // it's a check
+    },
+    // Now go through the settings for a new website. Previous steps created
+    // a frontend lang cookie which should not be used when redirecting to the
+    // frontend after the settings in the following steps.
+    {
+        content: "Go to backend settings",
+        trigger: 'body',
+        run: function () {
+            window.location.href = '/web#action=website.action_website_configuration';
+        }
+    },
+    {
+        content: "create a new website",
+        trigger: 'button[name="action_website_create_new"]',
+    },
+    {
+        content: "insert website name",
+        trigger: 'input[name="name"]',
+        run: 'text Website EN'
+    },
+    {
+        content: "validate the website creation modal",
+        trigger: 'button.btn-primary'
+    },
+    {
+        content: "make hover button appear",
+        trigger: '.o_theme_preview',
+        run: function () {
```

Comment:
Did you really applied that review to only the occurrence I commented, instead of all them? I wasn't especially even asking for a change but suggesting it, it was mostly to give you the info / share knowledge about it. (The more you see about it, the more you will think about it)

But partially applying it was not a possibility.. Either ignore it or apply it correctly (everywhere)